tools/Customization/DevHome.FileExplorerSourceControlIntegration/SourceControlProvider.cs

```diff
@@ -47,7 +49,7 @@ internal ILocalRepositoryProvider GetLocalProvider(string rootPath)
         var providerPtr = IntPtr.Zero;
         try
         {
-            var repositoryTracker = new RepositoryTracking(null);
+            repositoryTracker.ReloadRepositoryStoreIfChangesDetected();

```

Comment:
Seems odd that this reload is a separate call. Why wouldn't this be part of the implementation of `GetSourceControlProviderForRootPath`?